src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/validation/OcrValidator.java

```diff
@@ -172,4 +174,8 @@ private FormData toFormData(InputScannableItem doc) {
                 .collect(toList())
         );
     }
+
+    private String getFormType(InputScannableItem item) {
+        return item.documentType == InputDocumentType.SSCS1 ? "SSCS1" : item.documentSubtype;
```

Comment:
to match https://github.com/hmcts/sscs-bulk-scan/blob/0d1373b829466806fb271482a874d9e5c5549f38/src/main/java/uk/gov/hmcts/reform/sscs/domain/FormType.java